src/auth/db/datomic/credential.clj

```diff
@@ -0,0 +1,19 @@
+(ns auth.db.datomic.credential
+  (:require [schema.core :as s]
+            [auth.models.credential :as models.credential]
+            [common-labsoft.protocols.datomic :as protocols.datomic]
+            [common-labsoft.time :as time]
+            [common-labsoft.datomic.api :as datomic.api]))
+
+(s/defn new-credential! :- models.credential/Credential
+  [credential :- models.credential/Credential
+   datomic :- protocols.datomic/IDatomic]
+  (let [full-credential (assoc credential :credential/created-at (time/now))]
```

Comment:
Maybe we should extract this to a common function

Reply:
I think its well encapsulated here, it;s the only placed it's used

Reply:
But I mean, all entities will have created-at field, right?